pkgs/tools/misc/brotab/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, glibcLocales, python3 }:
```

I think python as a variable is easier to conceptualize. Just pass the version in all-packages.nix
```suggestion
{ stdenv, fetchFromGitHub, glibcLocales, python }:
```